ex01/Makefile

```diff
@@ -16,7 +18,7 @@ NAME				:= strlen_test
 all:				$(NAME)
 
 $(NAME):		$(OBJS)
-						$(CC) -o $(NAME) $(OBJS) $(LDFLAGS)
+						$(CC) $(FLAGS) -o $(NAME) $(OBJS) $(LDFLAGS)
```

Comment:
should we have CFLAGS here ?

Reply:
@emasiuk and me decided to add cflags here, as it can show additional warnings and notifications in test files and headers while compiling.

It can help to understand the proper use of funcs  and so on.
What do you think?

Reply:
I know I asked him to add thoses,
I'm saying you shouldn't need a FLAG and CFLAG var,
only CFLAG should work

Reply:
I'll do it tomorrow.